src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -607,6 +607,14 @@ public <T> T deepCopy(T object, Class<T> objectClass) {
         }
     }
 
+    public <T> T deepCopyArray(T object, TypeReference<T> typeReference) {
+        try {
```

Comment:
We have DeepCopy method already in this class. Use that.

Reply:
DeepCopy method is for object not list of array so I have created new one for array